ocaml/xapi/xapi_stunnel.mli

```diff
@@ -0,0 +1,23 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+(** Wrapper around the stunnel@xapi systemd service.
+    There exist scripts (e.g. xe-toolstack-restart) which also manipulate
+    the stunnel daemon but they do this directly (not via ocaml). *)
+
+val restart : __context:Context.t -> accept:string -> unit
+(** restart stunnel, possibly changing the config file *)
+
+val reload : unit -> unit
+(** reload (potentially updated) configuration *)
```

Comment:
As far as I can tell from the implementation, the only way to change the config is to call `restart`, which means calling `reload` wouldn't actually load a new config (this is probably correct - if you change the config completely sending a SIGHUP is probably not the right thing to do)

tldr; the purpose of reload is for when you've changed your trusted certs? the comment should probably talk about that rather than the config

Reply:
I think you're right yes. It's pre-existing code/comment, but I can add a commit to make it clearer now that I'm here.